sBTC SIP

```diff
@@ -0,0 +1,166 @@
+
+### Iterating Toward an Open Signer Network
+This SIP describes an iterative approach to sBTC development via a release plan that is simpler to implement and helps accelerate the sBTC release timeline, but does not include the complete feature set described in the original sBTC design documents.
+
+Thus, the rollout for sBTC will include two key phases:
+
+**Bootstrap:** In this phase an initial group of Signers selected through an open community governance process will be responsible for Signing sBTC transactions on the network. The initial group of Signers will be unchanged throughout the Bootstrap phase of the release schedule. During this phase, sBTC will have a distinct signer set (known as the “sBTC Bootstrap Signer”). And sBTC will not explicitly be integrated into the Stacks Proof of Transfer consensus. As a result, this release can activate without a hard fork.
+
+**Signer Rotation:** In this phase the full design of the sBTC system with an open and decentralized Signer set is reached, increasing decentralization after the system is further tested in the bootstrap phase. At this point, sBTC is integrated into the Proof of Transfer consensus and Signers receive rewards for fulfilling sBTC operations. This release does require a hard fork.
+
+#### Differences in the sBTC Bootstrap Phase:
+- sBTC Signer eligibility criteria will be confirmed through community governance vote. The eligibility criteria to become an sBTC signer is described below and will take into account Signer performance and availability. sBTC Signers that meet the criteria are confirmed by the sBTC Working Group. A subsequent release will implement an open and dynamically rotating signer set for each Proof of Transfer cycle, as described in the original sBTC design documents.
+- sBTC Bootstrap Signers are separate from Stacks Signers. In a subsequent sBTC release, and accompanying Stacks network hard fork, Stacks signers will be required to become sBTC signers. The bootstrapping period uses a separate subset of Stacks signers to secure the sBTC protocol and Signer operations are not explicitly linked with slashing of rewards.
+- sBTC deposits will be triggered via an API call. During the bootstrap phase sBTC deposits must be initiated via an API call to alert the signers to the presence of a deposit. After the bootstrapping phase users will have the option to alert the Signers of a deposit via a Stacks contract call, but the API will remain as a fee-less option to communicate with the Signers.
+
+These changes enable sBTC to be launched earlier and with less risk than the long term decentralized approach. The sBTC release with an open and dynamic signer set is expected to be released early 2025, which would enshrine the sBTC protocol into Stacks consensus.
```

Comment:
Do not -- I repeat -- do _NOT_ specify a release date.  First, no one on this earth knows when this iteration of sBTC will be ready to ship.  Second, the community -- not this SIP -- decides when a future sBTC activates if it's going to take a hard fork.  Third, given the high risks of building a resilient and robust two-way peg, the overriding release criterion for an sBTC system that is tied to consensus is always and forevermore going to be "when it's ready, and not a day sooner."

Reply:
+1 
i'm not sure of any alternative wording here, but agreed that mentioning a specific dates/timeframes should be discouraged. 
maybe focus on the fact that this will happen after the initial bootstrap phase, and what the criteria would be to move on to the next phase?